### PR TITLE
feat: browser analysics support attributes

### PR DESCRIPTION
**Pull request by iutx**
#### What this PR does / why we need it:
browser analysics support attributes

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=586748&iterationID=12783&type=TASK)

#### Specified Reviewers:

/assign  @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | browser analysics support attributes             |
| 🇨🇳 中文    |  浏览器数据分析支持属性字段            |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/apps/msp/apm/browser/consumer.go [hunk 1]
```diff
@@ -65,8 +71,11 @@ func (p *provider) invoke(key []byte, value []byte, topic *string, timestamp tim
 	metric.Tags["cid"] = parts[1]
 	metric.Tags["uid"] = parts[2]
 	metric.Tags["ip"] = parts[3]
-	if len(parts) > 5 {
-		metric.Tags["ai"] = parts[5]
+	if len(parts) > 5 && len(parts[5]) > 0 {
+		ai, err := base64.StdEncoding.DecodeString(parts[5])
```
Comment (sfwn) [inline internal/apps/msp/apm/browser/consumer.go:75]: you'd better add a comprehensive comment for `invoke` method to explain each part, especially for part 6 `ai`.

File: internal/apps/msp/apm/browser/consumer.go [hunk 2]
```diff
@@ -32,16 +31,12 @@ import (
 
 var (
 	other = "其他"
-	json  = jsoniter.ConfigCompatibleWithStandardLibrary
```
Comment (sfwn) [inline internal/apps/msp/apm/browser/consumer.go:35]: why remove it

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/apps/msp/apm/browser/consumer.go (hunk 2 above)
Comment (iutx) [reply internal/apps/msp/apm/browser/consumer.go:35]: > why remove it

unused variable.

**Review by sfwn — APPROVED** [review]

**Pull request merged**